### DISCUSSION TITLE
Feature Request: New Predefined Link Sources

### DISCUSSION THREAD

**Discussion by DerLinkman** [JulianPrieber/littlelink-custom#423] [Predefined Links]
Hello,

i want to ask if it is possible that someone could implement Spotify for Podcasters (previously Anchor), Google Podcasts and Amazon Music as a predefined site.

Yeah i guess it don't need any explaination why, as they don't exist currently.

**Comment by MagicLike** [comment]
> Hello,
> 
> i want to ask if it is possible that someone could implement Spotify for Podcasters (previously Anchor), Google Podcasts and Amazon Music as a predefined site.
> 
> Yeah i guess it don't need any explaination why, as they don't exist currently.

**Amazon Music already exists.
